Package.swift

```diff
@@ -0,0 +1,78 @@
+// swift-tools-version:4.2
+
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2019 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+
+import PackageDescription
+
+let package = Package(
+    name: "swift-tools-support-core",
+    products: [
+        .library(
+            name: "SwiftToolsSupport",
+            type: .dynamic,
+            targets: ["TSCBasic", "TSCUtility"]),
+        .library(
+            name: "SwiftToolsSupport-auto",
+            targets: ["TSCBasic", "TSCUtility"]),
+    ],
+    dependencies: [
+        
+    ],
+    targets: [
+        
+        // MARK: Tools support core targets
+        
+        .target(
+            /** Shim target to import missing C headers in Darwin and Glibc modulemap. */
```

Comment:
`/**` comments aren't actually valid here, can we use `//`?

Reply:
I copied the target declaration right from SwiftPM to match the comments there. Why are `/**` comments not valid here?

Reply:
`/**` are "doc comments", which need to get attached to a top-level decl like a class, or a function. You can't (to my knowledge) attach them to a parameter in a call. SwiftPM is wrong, but if that's what it does sure lets be consistent.

Reply:
TIL! I thought the only difference is that `/**` allows for multiline comments while `//` is one line only.

Reply:
*`/*`* allows for multiline.

`/**` is analogous to `///` in that they are Doxygen-derived conventions for annotating doc comments

Reply:
Oh true, thanks for for clarifying!